frontend/src/components/aboutUs.tsx

```diff
@@ -16,7 +16,7 @@ const AboutUs = () => {
             </div>
 
 
-            <p className="" style={{ fontSize: "48px", color: "#7D7D7D" }}>
+            <p className="text-2xl/relaxed md:text-5xl/relaxed" style={{color: "#7D7D7D" }}>
```

Comment:
I'm not sure if the classes `text-2xl/relaxed` and `md:text-5xl/relaxed` are proper tailwind styles because of the "/". One way to check if the style is getting properly applied is by hovering over the style and seeing a pop-up with the corresponding css styling.